### ISSUE TITLE
Sidebar alignment off in multi-monitor configuration

### ISSUE THREAD

**Issue by brian-avery** [PapyElGringo/material-awesome#28]
Love this awesomewm theme. It's great to work in and clean/minimal. Thanks for making it. 

I'm seeing an issue where the alignment of the sidebar is off when there are multiple monitors connected and the furthest to the left is not the primary. The sidebar seems to go into negative space when collapsed. 

Also, would it be possible to add a sidebar to each monitor? Awesomewm supports workspaces per monitor, but the theme only allows switching via the UI on the primary. Happy to move this to another issue if you'd like. The two just seemed related. 

![screenshot](https://user-images.githubusercontent.com/8484260/53059218-1722e100-3484-11e9-9a29-1e52386d51bc.jpeg)

**Comment by PapyElGringo** [comment]
1- Damn it ! I did design the workflow for vertical multi-screen (Like a notebook on the desk and a bigger screen above). But it's should work on horizontal setup also. Im gonna check what I can do about this!

2- It's totally doable and very easily ! But I made a choice to prevent the workspaces on secondary screen for the same reasons as Gnome. To keep simplicity. It's could be confusing to have the same workspaces again and again And you will have to keep in memory ( Of your own brain ) Where did you put this apps. For this reasons I made them like this.

But if you want to have them back just edit [panel.lua](https://github.com/PapyElGringo/material-awesome/blob/master/module/panel.lua) and only keep the **if** part of line 9 and remove the else part
